pantheon/src/test/java/tech/pegasys/pantheon/cli/PantheonCommandTest.java

```diff
@@ -1239,47 +1242,48 @@ public void maxpeersOptionMustBeUsed() {
   }
 
   @Test
-  public void fractionRemoteConnectionsAllowedOptionMustBeUsed() {
+  public void remoteConnectionsPercentageOptionMustBeUsed() {
 
-    final double fractionRemoteConnectionsAllowed = 0.12;
+    final int remoteConnectionsPercentage = 12;
     parseCommand(
-        "--limit-remote-wire-connections-enabled",
-        "--fraction-remote-connections-allowed",
-        String.valueOf(fractionRemoteConnectionsAllowed));
+        "--remote-connections-limit-enabled",
+        "--remote-connections-percentage",
+        String.valueOf(remoteConnectionsPercentage));
 
     verify(mockRunnerBuilder).fractionRemoteConnectionsAllowed(doubleArgumentCaptor.capture());
     verify(mockRunnerBuilder).build();
 
-    assertThat(doubleArgumentCaptor.getValue()).isEqualTo(fractionRemoteConnectionsAllowed);
+    assertThat(doubleArgumentCaptor.getValue())
+        .isEqualTo(
+            Fraction.fromPercentage(Percentage.fromInt(remoteConnectionsPercentage)).getValue());
 
     assertThat(commandOutput.toString()).isEmpty();
     assertThat(commandErrorOutput.toString()).isEmpty();
   }
 
   @Test
-  public void fractionRemoteConnectionsAllowedWithInvalidFormatMustFail() {
+  public void remoteConnectionsPercentageWithInvalidFormatMustFail() {
 
     parseCommand(
-        "--limit-remote-wire-connections-enabled",
-        "--fraction-remote-connections-allowed",
-        "not-a-fraction");
+        "--remote-connections-limit-enabled",
+        "--remote-connections-percentage",
+        "not-a-percentage");
     verifyZeroInteractions(mockRunnerBuilder);
     assertThat(commandOutput.toString()).isEmpty();
     assertThat(commandErrorOutput.toString())
         .contains(
-            "Invalid value for option '--fraction-remote-connections-allowed': cannot convert 'not-a-fraction' to Fraction");
+            "Invalid value for option '--remote-connections-percentage': cannot convert 'not-a-percentage' to Percentage");
```

Comment:
Question - where does this phrase "cannot convert 'not-a-percentage'" come from?  Is that generated internally by PicoCLI?  If we add a custom error message to the exception that's thrown can we get a better error message?

Reply:
Ok, will do the same logic applied for RPC APIs conversion.

Reply:
done.